src/html/logo.html

```diff
@@ -1,10 +1,10 @@
 <!-- section -->
 <section id="logo" class="section">
-  <h1 class="h1">Logo</h1>
-  <h2 class="h2">Icon</h2>
+  <h1 class="h1 weight-strong">Logo</h1>
+  <h2 class="h2 weight-strong">Icon</h2>
   <div class="figure-wrapper">
-    <figure class="figure figure-logo-icon">
-      <img class="img" src="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.svg" width="190px" alt="Lockup Purple">
+    <figure class="grid-block figure vertical shrink figure-logo-icon">
+      <img class="img" src="images/logo/pink-icon/RGB/runnable-icon-pink-rgb.svg" width="190px" height="190px" alt="Lockup Purple">
```

Comment:
This looks better now but you still need to remove "px" from all your height and width values. So for example this line should be `width="190" height="190"` instead of `width="190px" height="190px"`